daemon/cmd/daemon_main.go

```diff
@@ -1451,6 +1452,21 @@ func (d *Daemon) instantiateAPI() *restapi.CiliumAPI {
 	return restAPI
 }
 
+func initSockmapOption() {
+	if !option.Config.SockopsEnable {
+		return
+	}
+	if probes.NewProbeManager().GetMapTypes().HaveSockhashMapType {
+		k := probes.NewProbeManager().GetHelpers("sock_ops")
+		h := probes.NewProbeManager().GetHelpers("sk_msg")
```

Comment:
```suggestion
		pm := probes.NewProbeManager()
		k := pm.GetHelpers("sock_ops")
		h := pm.GetHelpers("sk_msg")
```

Not an issue because probe results are cached, but suggestion for next time: don't create more probe managers than necessary :)